src/Dapr.Actors/Runtime/ActorConfig.cs

```diff
@@ -0,0 +1,116 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Actors.Runtime
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Represents Dapr actor configuration for this app.  See
+    /// https://github.com/dapr/docs/blob/master/reference/api/actors_api.md
+    /// </summary>
+    public sealed class ActorConfig
```

Comment:
rename to ActorSettings

Reply:
Is this a convention?  "Config" conveys the meaning of this object and it more closely resembles the name of the endpoint (which is not a goal, but it's good when it can).

Reply:
Not a convention, keeping the naming in line with SF actors.